app/routers/client.py

```diff
@@ -80,44 +80,69 @@ async def complete_client_visit(
 
 @router_client.post("/delete_clients_queue", response_model=str, tags=["Client"])
 async def delete_client_from_queue(
-    client_data: Client, doctor: Doctor = Depends(get_current_doctor)
+    client_data: ClientQueue, doctor: Doctor = Depends(get_current_doctor)
 ):
     """Delete client from queue"""
     service = QueueService()
     service.delete_client_from_queue(client_data, doctor)
     return "ok"
 
 
-@router_client.get("/queue", response_model=List[Client], tags=["Client"])
+@router_client.get("/queue", response_model=List[ClientQueue], tags=["Client"])
 def get_queue(doctor: Doctor = Depends(get_current_doctor)):
     """Show clients in queue"""
-    reception = Reception.query.filter(Reception.date == datetime.date.today()).first()
+    today = datetime.date.today()
+    reception = Reception.query.filter(Reception.date == today).first()
     if not reception:
         reception = Reception(doctor_id=doctor.id).save(True)
 
     queue_members = QueueMemberDB.query.filter(
         and_(
             QueueMemberDB.reception_id == reception.id,
-            QueueMemberDB.visit_id == None,  # noqa E711
-            QueueMemberDB.canceled == False,
+            QueueMemberDB.canceled == False,  # noqa E712
         )
     ).all()
 
     members = [
-        {"client": member.client, "canceled": member.canceled}
+        {
+            "client": member.client,
+            "canceled": member.canceled,
+            "place_in_queue": member.place_in_queue,
+        }
         for member in queue_members
     ]
 
     members_without_complete_visit = []
     for member in members:
+        client_member = {
+            "api_key": member["client"].client_info["api_key"],
+            "email": member["client"].client_info["email"],
+            "first_name": member["client"].client_info["firstName"],
+            "id": member["client"].client_info["id"],
+            "last_name": member["client"].client_info["lastName"],
+            "phone": member["client"].client_info["phone"],
```

Comment:
I think it would be great to output this part of the code ( member["client"].client_info ) to a separate variable and assign values to keys from it (e.g member_info = member["client"].client_info;  member_info["phone"]) or smth like this). At least this will increase the readability of the code

Reply:
Thanks for the advice, and I will )